index.html

```diff
@@ -622,7 +622,7 @@ <h4 id="cert-cost">What about certificate costs?</h4>
       <p>You can get free certificates for any use from <a href="https://letsencrypt.org/">Let’s Encrypt</a>. If you need EV verification, then you will have to pay a bit extra. Use your favorite search engine to look for and evaluate the available options. The security and integrity of your visitors' data is worth every penny!</p>
 
       <h4 id="ecc">What about Elliptic Curve Cryptography (ECC) certificates?</h4>
-      <p>ECC certificates offer <a href="https://casecurity.org/2014/06/10/benefits-of-elliptic-curve-cryptography/">stronger security and smaller certificates</a> - e.g. a 256-bit ECC key is equivalent to a 3072-bit RSA key. However, many clients do not support <a href="https://en.wikipedia.org/wiki/Elliptic_Curve_DSA">ECDSA</a>, which means that the server should support both RSA and ECDSA, and some popular servers do not support dual certificate deployments. Consult the documentation of your server to see if RSA+ECDSA is a supported option.</p>
+      <p>ECC certificates offer <a href="https://casecurity.org/2014/06/10/benefits-of-elliptic-curve-cryptography/">stronger security and smaller certificates</a> - e.g. a 256-bit ECC key is equivalent to a 3072-bit RSA key.  <a href="https://en.wikipedia.org/wiki/Elliptic_Curve_DSA">ECDSA</a> certificates are a great choice for modern TLS clients such as web browsers to use, however you may need to support <a href="https://support.globalsign.com/ssl/ssl-certificates-life-cycle/ecc-compatibility">legacy clients</a> (OS/software often from 2008 or earlier) that lack ECDSA support - such as within an enterprise environment. It’s possible to <a href="https://scotthelme.co.uk/hybrid-rsa-and-ecdsa-certificates-with-nginx/">provide both certificates with RSA as a fallback</a> (often referred to as dual or hybrid certificates). Consult the documentation of your server to see if RSA+ECDSA is a supported option.</p>
```

Comment:
Thanks for drafting this! 

Q: Should we make a stronger recommendation for ECDSA?

Perhaps a shorter version...

_ECDSA certificates are recommended for modern TLS clients, such as web browsers, but if you need to support legacy clients (e.g., OS/software from 2008 or earlier often found in enterprise environments) that lack ECDSA support, you can provide a hybrid certificate that includes an RSA fallback. Consult the documentation..._

Reply:
> Q: Should we make a stronger recommendation for ECDSA?

I'm not an expert, but I don't see any major reason to do so. Browser support at least is fairly limited, only NIST curves AFAIK, these both have a concern of potential NSA backdoors, but nothing that has been found/proven thus far. EdDSA is the ideal choice whenever that becomes available.

P-256 is equivalent to 128-bits of entropy, akin to RSA-3072. I'm not sure if it applies to others, but in my case I found a notable performance hit with changing to P-384:

```
openssl speed rsa2048 rsa3072 rsa4096 ecdsap256 ecdsap384 ecdsap521
```

|                                           | sign            | verify          | sign/s    | verify/s   |
|------------------------------------|----------------|-----------------|------------|-------------|
| rsa 2048 bits                      | 0.000619s |  0.000019s | 1614.6   | 53725.7 |
| rsa 3072 bits                      | 0.001865s | 0.000038s  | 536.2     | 26465.5 |
| rsa 4096 bits                      | 0.004187s | 0.000065s  | 238.8     | 15322.7 |
| 256 bits ecdsa (nistp256)  | 0.0000s     | 0.0001s      | 40769.7 | 13987.6 |
| 384 bits ecdsa (nistp384)  | 0.0010s     | 0.0008s      | 1005.4   | 1240.7   |
| 521 bits ecdsa (nistp521)  | 0.0003s     | 0.0006s      | 3135.3   | 1577.9   |

The degradation was significantly larger than RSA-2048 vs RSA-4096 for me. P-384 still had over 4x the performance in signatures/sec over RSA-4096, but compared to P-256, it was 40k vs 1k sig/sec for me, with verifications/sec being 14k vs 1.2k (RSA-4096 managed 15k), oddly P-521 performed better with 3k sig/sec and 1.5k verifications/sec, Chrome removed support for it years ago however.

`sign/s` is server side, while `verify/s` would be on the client. Although this isn't an area of expertise for me to really advise strongly on.

---

> Perhaps a shorter version...

Updated the PR, you omitted the initial sentence? That still seemed useful, did you want it removed?